frontend/class-twitter.php

```diff
@@ -187,7 +187,7 @@ private function single_description( $post_id = 0 ) {
 		$meta_desc = trim( WPSEO_Meta::get_value( 'twitter-description', $post_id ) );
 
 		if ( is_string( $meta_desc ) && '' !== $meta_desc ) {
-			return $meta_desc;
+			return wpseo_replace_vars( $meta_desc, $GLOBALS['wp_query']->get_queried_object() );
```

Comment:
Can't `get_queried_object()` be used directly instead of calling it from the `$GLOBALS`?